Tools/clinic/clinic.py

```diff
@@ -3657,15 +3732,33 @@ def converter_init(self, *, bitwise: bool = False) -> None:
             self.add_include('pycore_long.h',
                              '_PyLong_UnsignedShort_Converter()')
 
-    def parse_arg(self, argname: str, displayname: str) -> str | None:
+    def parse_arg(self, argname: str, displayname: str, *, limited_capi: bool) -> str | None:
         if self.format_unit == 'H':
-            return """
+            return self.format_code("""
                 {paramname} = (unsigned short)PyLong_AsUnsignedLongMask({argname});
                 if ({paramname} == (unsigned short)-1 && PyErr_Occurred()) {{{{
                     goto exit;
                 }}}}
-                """.format(argname=argname, paramname=self.parser_name)
-        return super().parse_arg(argname, displayname)
+                """,
+                argname=argname)
+        if not limited_capi:
+            return super().parse_arg(argname, displayname, limited_capi=limited_capi)
+        # TODO: Raise ValueError for negative integer
```

Comment:
Are the introduced TODO comments for later, or do you intend to resolve them before this is landed?

Reply:
#74020 could resolve it. Without it we can only record the fact of minor difference. But these converters are not used much (actually, only one of them is used in one place), so it is not too important problem.